Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -489,33 +491,33 @@ window.__firefox__.execute(function($) {
     }
     // Find selectors we have a rule index for
     const rulesToRemove = Array.from(selectors)
-      .map(function (selector) { return CC.allSelectorsToRules.get(selector) })
+      .map(function (selector) { return CC.allHideSelectorsToRules.get(selector) })
       .filter(function (i) { return i !== undefined })
       .sort()
       .reverse()
 
     // Delete the rules
-    const lastIdx = CC.allSelectorsToRules.size - 1
+    const lastIdx = CC.allHideSelectorsToRules.size - 1
     // to deal with removing rules
     
     for (let _i = 0, rulesToRemove1 = rulesToRemove; _i < rulesToRemove1.length; _i++) {
```

Comment:
similarly, maybe a better name than `rulesToRemove1` and `rulesToRemove`

Reply:
This was actually removed. I think the code here was just some ugly typescript conversion.